Mathlib/Algebra/Free.lean

```diff
@@ -169,7 +169,7 @@ protected def recOnPure {C : FreeMagma α → Sort l} (x) (ih1 : ∀ x, C (pure
 #align free_magma.rec_on_pure FreeMagma.recOnPure
 
 -- Porting note (#10675): dsimp can not prove this
-@[to_additive (attr := simp, nolint simpNF)]
+@[to_additive (attr := simp)]
```

Comment:
Can you follow up to remove all the comments that explain why we used `nolint simpNF`, like this one?